src/Roll.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+function Roll(){
+  this._remainingPins = 10;
+  this._knockedDownPins = 0;
+  this._rollNumber = 1;
+  this._isSpare = false;
+  this._isStrike = false;
+}
+
+Roll.prototype.pinNumber = function(){
+  return this._remainingPins;
+};
+
+Roll.prototype.rollNumber = function(){
+  return this._rollNumber;
+};
+
+Roll.prototype.isSpare = function(){
+  return this._isSpare;
+};
+
+Roll.prototype.isStrike = function(){
+  return this._isStrike;
+};
+
+Roll.prototype.knockedDownPins = function(){
+  if (this._rollNumber === 1) {this.knockedDownPinsFirstRoll();
+    return this._knockedDownPins;}
+  if (this._rollNumber === 2) {this.knockedDownPinsSecondRoll();
+    return this._knockedDownPins;}
+};
+
+Roll.prototype.knockedDownPinsFirstRoll = function(){
```

Comment:
This function has too many statements. (28)<br>This function's cyclomatic complexity is too high. (13)